Geometry/HcalAlgo/test/runTest.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
@bsunanda what is the purpose of this file? As it is, it doesn't run as a unit test, although it could if a BuildFile is instructed for this.

Reply:
@bsunanda could you please address my question?

Reply:
I am adding what could be unit test for geometry algorithms. I am doing this because I am in the position of porting these algorithms to DD4Hep algorithms.

Reply:
@bsunanda it might be good to instrument the code so as to really run it as unit test, so far it does not and it is just a standalone test code